week-1/exercises/random.js

```diff
@@ -5,5 +5,7 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
+// Ans. The expression is trying to multiply a random number from 1 to 100 with 2 and then adding 1 to the value
```

Comment:
I'm not sure I follow this explanation Areeb, I'm confused about where the `2` would come from. Take a look at the range of numbers that `Math.random()` could produce, which might help to explain why we need to do so many calculations with a `minimum` and `maximum` value

Reply:
I have tried to explain it again, is it right this time or still not there?